.circleci/config.yml

```diff
@@ -29,6 +29,16 @@ jobs:
           root: .
           paths: site
 
+  build_docs_pdf:
+    docker:
+      - image: danteev/texlive
+    steps:
+      - checkout
+      - run: 
+        command: |
+         cd pdf_build_src
+         sh build_pdf.sh
```

Comment:
you may want to add 

```
      - store_artifacts:
          path: rendered_pdf
```

see line 26 --> that would store a pdf in the "artifacts tab" (if the build succeeds) of the CircleCI output, see for example these files: https://circleci.com/gh/bids-standard/bids-specification/1799#artifacts/containers/0

Reply:
Thank you! I have no prior experience with circleci so not really sure how to fix the following error: 
`Build-agent version 1.0.27691-0016e3a2 (2020-02-20T13:14:44+0000)

Configuration errors: 2 errors occurred:

* Error migrating config to version 2: 2 errors occurred:

* in job 'build_docs_pdf': step was not in the right format (unable to find step's name/details)
* in job 'build_docs_pdf': step was not in the right format (unable to find step's name/details)
* In step 2 definition: Step

  - command: cd pdf_build_src
sh build_pdf.sh

    run: map[]

has incorrect indentation,it should be formatted like:

  - step:
      option1: value
      option2: value`

I fixed the indentation but still getting the same error. Also, I'm not clear on why `build_docs` job is failing even though it was succeeding before adding `build_docs_pdf` job. I'd appreciate help on this one.